webpack/renderer.ts

```diff
@@ -143,15 +151,19 @@ export function fontsLoaderWebpackRules(): webpack.RuleSetRule[] {
   ];
 }
 
+export interface CssModulesWebpackRuleOptions {
+  styleLoader?: string;
+}
+
 /**
  * Import CSS or SASS styles with modules support (*.module.scss)
- * @param {string} styleLoader
  */
-export function cssModulesWebpackRule(
-  {
-    styleLoader = vars.isDevelopment ? "style-loader" : MiniCssExtractPlugin.loader,
-  } = {}): webpack.RuleSetRule {
-  const { isDevelopment, sassCommonVars } = vars;
+export function cssModulesWebpackRule({ styleLoader }: CssModulesWebpackRuleOptions = {}): webpack.RuleSetRule {
+  styleLoader ??= (
+    isDevelopment
+      ? "style-loader"
+      : MiniCssExtractPlugin.loader
+  );
```

Comment:
Why so many extra lines? _(harder to read whole expression than just in a single line)_

Reply:
I prefer this style for the `?:` operator because it makes it easier to notice both options instead of having to search along a single line for the `:`. This style has both options clearly demarked but I do admit this is a style thing.

Reply:
Ok, maybe then just remove wrapping in brackets?

Reply:
Sure I can, I prefer having it on separate lines but it isn't that big of a deal.